.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+# Issue information
+
+> Define issue title **short** and **most specific** to the problem.
+
+> Tag each new issue with relevant **LABELS** (*Bug*, *Improvement*, etc.).
+
+## **BUG** report
+
+### Environment
+
+- Apache Camel Language Server release version date: e.g. "**1.0.0-SNAPSHOT**, **Jun-04**" ?
```

Comment:
I think users won't know this version. They will know only the version of VS Code extension

Reply:
so better to use something like: 

- Language Support for Apache Camel version: e.g. "**v0.0.13**" ?

Reply:
yes